app/src/components/components/ButtonConfigurator/ButtonConfigurator.tsx

```diff
@@ -35,7 +36,7 @@ export const ButtonConfigurator = () => {
           label: name,
         }))}
         onChange={(item) => {
-          setVariant(item.value as HvButtonVariant);
+          setVariant((item as HvListValue).value as HvButtonVariant);
```

Comment:
We definitely need a proper Select element...

Reply:
Agreed 🙏